src/JustEat.StatsD/StatsDMessageFormatter.cs

```diff
@@ -5,7 +5,9 @@
 
 namespace JustEat.StatsD
 {
+#if NETDESKTOP
```

Comment:
You could use the standard ```NET45``` & ```NETSTANDARD1_6``` compilation symbols rather than define your own. Or a lot of people define their own per usage, so here would be something like ```#if BINARY_SERIALATION``` and then in the project.json you can mix and match what you need per TFM, although this arguably has less value now that we have netstandard.